apps/barista/src/components/color-grid/color-grid.ts

```diff
@@ -31,30 +31,44 @@ export class BaColorGrid implements AfterContentInit {
     this._colorInput = value;
   }
 
-  /** @internal name of the colorgroup */
-  _colorname: string;
+  @Input()
+  get colorList(): string {
+    return this._colorList;
+  }
+  set colorList(value: string) {
+    this._colorList = value;
```

Comment:
when you are not modifying the passed in value or calling any other methods, there is no need for a getter/setter.
A getter setter is only needed when you do additional stuff like the uppercasing in the color property.
Please change it to a normal property.